src/components/BlogPost/BlogPost.js

```diff
@@ -46,24 +100,51 @@ function BlogPost(props) {
           <Grid>{blogSideDate}</Grid>
           {blogSideTags}
         </Grid>
-        <Grid item xs={9}>
+        <Grid item lg={9} xs={12}>
           <Link
+            className={width === 'xs' ? classes.linkSmall : classes.link}
```

Comment:
See how you use ```width === 'xs'``` here, line 117, and line 125? To make it more efficient and keeping to the coding (Don’t Repeat Yourself) principle DRY,  you could make a constant on line 95 ```const smallScreen = width === 'xs'``` and then use that constant like ```className={smallScreen ? classes.linkSmall : classes.link}```